face_api_dataset/dataset.py

```diff
@@ -955,3 +1001,47 @@ def _read_landmarks_3d(cls, info: dict, mdt: Modality) -> Dict[Id, Landmark_3D]:
             landmarks[lmk_name] = tuple(landmark["camera_space_pos"])
 
         return landmarks
+
```

Comment:
Following functions are not used anywhere. I guess it was intended and this PR is still wip.